common.sh

```diff
@@ -31,9 +31,9 @@ command -v gcloud >/dev/null 2>&1 || { \
 # bastion set up
 BASTION_INSTANCE_NAME=gke-demo-bastion
 # set to jenkins if there is no $USER
-USER=$(whoami)
-[[ "${USER}" == "root" ]] && export USER=jenkins
-echo "user is: $USER"
+# USER=$(whoami)
```

Comment:
I am wondering if we enable tty if we need this ... I will merge but something to backlog